requirements/base.txt

```diff
@@ -10,7 +10,7 @@ djangorestframework-csv==1.4.1
 tqdm==4.8.3                     # progress bars
 requests==2.10.0
 https://github.com/cherrypy/cherrypy/archive/v6.2.0.zip#egg=cherrypy
-https://github.com/fle-internal/django-q/archive/934d557d77ded18c4a73a702905a6f8fe932f97b.zip#egg=django-q
+https://github.com/benjaoming/django-q/archive/fc5244b30c98035cb72db12186471a1c6335d364.zip#egg=django-q
```

Comment:
Ah yes, I do that sometimes, too - great idea!

Regarding this whole django-q deal, I just got admin rights for fle-internal yesterday that's why I couldn't progress more than to just create yet another repo. It's a mess. I'll clean up our own fle-internal repo so it aligns with the upstream `master` and will create feature branches and PRs for our own stuff so it goes back upstream.